azure-pipelines-official.yml

```diff
@@ -170,14 +170,18 @@ stages:
       condition: and(succeeded(), eq(variables['PRNumber'], 'default'))
 
     # Publish OptProf configuration files
+    # The env variable is required to enable cross account access using PAT (dnceng -> devdiv)
     - task: ms-vscs-artifact.build-tasks.artifactDropTask-1.artifactDropTask@0
+      env:
+        ARTIFACTSERVICES_DROP_PAT: $(_DevDivDropAccessToken)
       inputs:
         dropServiceURI: 'https://devdiv.artifacts.visualstudio.com'
         buildNumber: 'ProfilingInputs/DevDiv/$(Build.Repository.Name)/$(SourceBranchName)/$(Build.BuildNumber)'
         sourcePath: '$(Build.SourcesDirectory)\artifacts\OptProf\$(BuildConfiguration)\Data'
         toLowerCase: false
         usePat: false
         retentionDays: 90
+        detailedLog: true
```

Comment:
did you mean to leave this here? (probably fine but looks like it was there to debug)

Reply:
Yea it was for debug, but I think 100 extra lines of drop.exe log is OK :)